x11-libs/libxaw/libxaw-1.0.13.recipe

```diff
@@ -0,0 +1,117 @@
+SUMMARY="X Athena Widget library"
+DESCRIPTION="X Athena Widgets, or Xaw, is a widget set based on the X Toolkit \
+Intrinsics (Xt) Library. Xaw & Xt form together what is known as the X Toolkit."
+HOMEPAGE="https://www.x.org/releases/individual/lib/"
+COPYRIGHT="1985-1990, 1994, 1998 The Open Group
+	2003-2004 Roland Mainz
+	1998-1999 The XFree86 Project, Inc.
+	1991 OMRON Corporation
+	1985-1998 Digital Equipment Corporation
+	1989 Prentice Hall
+	1985-1989, 1991, 1994 X Consortium"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://www.x.org/releases/individual/lib/libXaw-$portVersion.tar.bz2"
+CHECKSUM_SHA256="8ef8067312571292ccc2bbe94c41109dcf022ea5a4ec71656a83d8cce9edb0cd"
+SOURCE_DIR="libXaw-$portVersion"
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libXaw6Version="6.0.1"
+libXaw7Version="7.0.0"
+
+libXaw6VersionCompat="$libXaw6Version compat >= ${libXaw6Version%%.*}"
+libXaw7VersionCompat="$libXaw7Version compat >= ${libXaw7Version%%.*}"
+
+PROVIDES="
+	libxaw$secondaryArchSuffix = $portVersion
+	lib:libxaw6$secondaryArchSuffix = $libXaw6VersionCompat
+	lib:libxaw7$secondaryArchSuffix = $libXaw7VersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libX11$secondaryArchSuffix
+	lib:libxext$secondaryArchSuffix
+	lib:libXmu$secondaryArchSuffix
+	lib:libXpm$secondaryArchSuffix
+	lib:libXt$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libxaw${secondaryArchSuffix}_devel = $portVersion
+	devel:libXaw6$secondaryArchSuffix = $libXaw6VersionCompat
+	devel:libXaw7$secondaryArchSuffix = $libXaw7VersionCompat
+	"
+REQUIRES_devel="
+	libxaw$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:kbproto$secondaryArchSuffix
+	devel:libICE$secondaryArchSuffix
+	devel:libpthread_stubs$secondaryArchSuffix
+	devel:libX11$secondaryArchSuffix
+	devel:libxext$secondaryArchSuffix
+	devel:libXmu$secondaryArchSuffix
+	devel:libXpm$secondaryArchSuffix
+	devel:libXt$secondaryArchSuffix
+	devel:libSM$secondaryArchSuffix
+	devel:xextproto$secondaryArchSuffix
+	devel:xproto$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	devel:util_macros$secondaryArchSuffix
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake
+	cmd:autoheader
+	cmd:gcc$secondaryArchSuffix
+	cmd:grep
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:makeinfo
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:sed
+	"
+
+defineDebugInfoPackage libxaw$secondaryArchSuffix \
+	$libDir/libXaw6.so.$libXaw6Version \
+	$libDir/libXaw7.so.$libXaw7Version
+
+BUILD()
+{
+	autoreconf -vfi
+	runConfigure ./configure
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	rm -f $libDir/libXaw*.la
+
+	# remove unwanted symlinks
+	rm -f $libDir/libXaw.so*
+
+	prepareInstalledDevelLibs \
+		libXaw6 \
+		libXaw7
+	fixPkgconfig
+
+	# add the appropriate symlinks
+	ln -s libXaw6.so.$libXaw6Version $libDir/libXaw6.so.6
+	ln -s libXaw7.so.$libXaw7Version $libDir/libXaw7.so.7
```

Comment:
Since the soname is

     0x000000000000000e (SONAME)             Library soname: [libXaw.so.7]

there's no need for these

```diff
-	ln -s libXaw6.so.$libXaw6Version $libDir/libXaw6.so.6
-	ln -s libXaw7.so.$libXaw7Version $libDir/libXaw7.so.7